### PR TITLE
Chapter 12

### PR DESCRIPTION
**Pull request by balqis277**
ready

### CODE REVIEW

**Comment by balqis277** [comment]
ready

**Comment by balqis277** [comment]
ready

**Comment by balqis277** [comment]
ready

File: app/views/password_resets/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<% provide :title, "Forgot password" %>
+<h1>Forgot password</h1>
+  <div class="row">
+    <div class="col-md-6 col-md-offset-3">
+      <%= form_for :password_reset, url: password_resets_path do |f| %>
+      <%= f.label :email %>
+      <%= f.email_field :email, class: "form-control" %>
+      <%= f.submit "Submit", class: "btn btn-primary" %>
+    <% end %>
```
Comment (anlvq-1058) [inline app/views/password_resets/new.html.erb:9]: indent please

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by balqis277** [comment]
ready

**Comment by balqis277** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged
You can merge this pull request

**Pull request merged**